example/react-native.config.js

```diff
@@ -5,6 +5,11 @@ if (process.argv.includes("--config=metro.config.macos.js")) {
 } else {
   module.exports = {
     project: {
+      android: {
+        sourceDir: "android",
+        manifestPath:
+          "../node_modules/react-native/template/android/app/src/main/AndroidManifest.xml",
```

Comment:
Was this necessary? I would rather check-in the manifest file into example.

Reply:
It's mostly to keep the `android` folder clean and as close to a real world package as possible:
```sh
% find android
android
android/gradle
android/gradle/wrapper
android/gradle/wrapper/gradle-wrapper.jar
android/gradle/wrapper/gradle-wrapper.properties
android/gradlew
android/build.gradle
android/gradle.properties
android/gradlew.bat
android/settings.gradle
```
Should I check it in?

Reply:
Actually, it looks like I can do without? Don't recall why it was needed in the first place ...

Reply:
When I was working with the example there was no need for the manifest. Removing it would be ideal or alternatively checking it in. Otherwise RN CLI tools will have a ref to a manifest while AS doesn't, which can cause problems when working with it from IDE.

Reply:
It's removed.